Resources/Private/Templates/FormBuilder/Index.html

```diff
@@ -8,6 +8,8 @@
 			<strong><f:translate id="formBuilder.index.backend.addFormElements" package="CRON.FormBuilder" /></strong>
 		</f:if>
 		<ts:render path="elements" typoScriptPackageKey="{tsPackageKey}" context="{node: elements}"/>
+		<input style="display: none" type="text" class="honeypot" name="--cron_formbuilder-plugin[data][subject]" value="">
+		<input style="display: none" type="text" class="honeypot" name="--cron_formbuilder-plugin[data][phone]" value="{attributes.timehash}">
```

Comment:
@AndreEtienne was wird denn hier im name-Attribut gerendert?
Dass ein Benutzer des Formbuilders einen echten Input mit name="phone" hinzufügt, ist schon nicht unwahrscheinlich. Dadurch gäbe es einen Konflikt, wenn zwei Inputs mit name="phone" im selben Formular existieren.

Reply:
Ja da gebe ich dir recht ich habe das jetzt in der validator class gändert das hier nichts mehr passieren kann.